presto-native-execution/presto_cpp/main/PeriodicTaskManager.h

```diff
@@ -89,6 +89,8 @@ class PeriodicTaskManager {
   const std::unordered_map<
       std::string,
       std::shared_ptr<velox::connector::Connector>>& connectors_;
+
+  int64_t userCpuTimeMicrosOld{0};
```

Comment:
```
s/int64_t userCpuTimeMicrosOld{0};/int64_t lastUserCpuTimeUs_;/
```

Reply:
void PeriodicTaskManager::addOperatingSystemStatsTask() {
  scheduler_.addFunction([this](){ updateOperatingSystemStats(); });
}